manifest.json

```diff
@@ -5,14 +5,14 @@
     "description": {
         "en": "A flexible event/agent & automation system "
     },
-    "version": "28.01.2021~ynh1",
+    "version": "0.4.0~ynh1",
```

Comment:
Eh I guess the upstream may have an actual version numbering scheme now ? But unfortunately moving from "28.01" to "0.4" means that Yunohost won't propose any upgrade because it thinks it's a lower version ... I dunno how big of an issue this is, maybe there aren't so many users for this app, idk

Or an hackish dirty way is to use something like `100.0.4.0~ynh1` instead but meh